gef.py

```diff
@@ -4789,7 +4794,7 @@ def create_or_edit_structure(self, mod_name, struct_name):
         else:
             info("Editing '{:s}'".format(fullname))
 
-        cmd = os.getenv("EDITOR").split() if os.getenv("EDITOR") else ["nano",]
+        cmd = (os.getenv("EDITOR") or "nano").split()
```

Comment:
Although this can be accepted, as it is cleaner, it is unrelated to this PR, therefore making it hard for us to track. Please make a different PR.